x/staking/querier/querier_test.go

```diff
@@ -134,19 +132,28 @@ func TestQueryValidators(t *testing.T) {
 	// Query Validators
 	queriedValidators := keeper.GetValidators(ctx, params.MaxValidators)
 
-	res, err := queryValidators(ctx, cdc, keeper)
+	queryValsParams := NewQueryValidatorsParams(1, int(params.MaxValidators), sdk.BondStatusUnbonded)
```

Comment:
can we add another test for the `BondStatusUnbonding` ?

Reply:
Sure, I can see if I can manually set their status.

Reply:
Updated.